kentik_api_library/kentik_api/synthetics/api_connector.py

```diff
@@ -16,7 +16,8 @@ class APISyntheticsConnector:
     Allows sending authorized grpc requests to Kentik Synthetics API
     """
 
-    def __init__(self, api_url: str, auth_email: str, auth_token: str, options: Tuple[Tuple[str, Any]] = ()):
+    def __init__(self, api_url: str, auth_email: str, auth_token: str,
```

Comment:
Instead of ignoring, you can declare it as List[Tuple[str, Any]] and later cast to tuple: `self._options = tuple(options)`

Reply:
Good idea. Done.